### ISSUE TITLE
Use ${{ github.token }} in our dogfood test

### ISSUE THREAD

**Issue by jcbhmr** [spenserblack/actions-wiki#22]
https://github.com/spenserblack/actions-wiki/blob/53e179fdd89d04a25a96c4abcaa0ade4da38f1b4/.github/workflows/update-wiki.yml#L25

**Comment by spenserblack** [comment]
Sounds good!

I'll respond to https://github.com/spenserblack/actions-wiki/issues/1#issuecomment-1470780865 here since I think it's relevant to this issue:

If the format is `${{ github.actor }}:${{ github.token }}@...`, then I think we add an input for user, defaulting to `${{ github.actor }}`, and use `${{ inputs.user }}:${{ inputs.token }}` (something besides `user` might be better). AFAIK the "user" for fine-grained tokens is `oauth2`, so this could also add support for that.

**Comment by jcbhmr** [comment]
@spenserblack I don't know if the user even matters? Maybe it does, maybe it doesn't... 🤔 something to test!

**Comment by spenserblack** [comment]
At the very least, we can say "if you're using a fine-grained token, use `user: oauth2`," which I believe to be true.

**Comment by jcbhmr** [comment]
I meant more so that you might be able to just use a dummy like `https://dummy:$TOKEN@github.com/user/repo.wiki.git`

**Comment by jcbhmr** [comment]
@spenserblack A dummy user works! You might even be able to get away with `https://:$TOKEN@github.com/user/repo.wiki.git` 👈 empty username

![image](https://user-images.githubusercontent.com/61068799/225510784-5be38cd4-fa11-4ece-ab41-367a12151d96.png)

![image](https://user-images.githubusercontent.com/61068799/225510799-de7ad78b-b979-489e-83d7-35a9c9b40f5a.png)

https://github.com/jcbhmr/.github/actions/runs/4433400235/jobs/7778395149

So no need for a "user" field at all, unless there's something about the weird OAuth2 tokens that you were referring to? Or are those agnostic to username as well?

**Comment by jcbhmr** [comment]
Nope, an empty username FAILS but the "dummy:${{ github.token }}" works!

![image](https://user-images.githubusercontent.com/61068799/225511151-e26d7efb-26be-4c40-a9cf-2aa1d10945f6.png)

**Comment by spenserblack** [comment]
That's interesting, I never expected that to work. To clarify, are you using a dummy user with a classic token, or a fine-grained token? Based on what I've read (i.e. stumbled on in Stackoverflow), fine-grained tokens do require the user to be `oauth2`.

**Comment by jcbhmr** [comment]
I'm using whatever token the `${{ github.token }}` value is. Which I assume is an auto-generated temporary Fine-Grained© GitHub® Access Token™

![image](https://user-images.githubusercontent.com/61068799/225727685-a0a4ab21-d2a0-4b2d-92bd-9baba5a7b8f5.png)

and it works!
